client/src/main/java/com/paypal/selion/reports/reporter/html/HtmlReporterListener.java

```diff
@@ -300,11 +300,13 @@ private String getContent(ITestResult result) {
                     // Attaching ralogId to each of the page title.
                     if ((logLine.getHref() != null) && (logLine.getHref().length() > 1)) {
                         htmllog = "<a href='../" + logLine.getHref() + "' title='" + logLine.getLocation() + "' >"
-                                + logLine.getMsg() + "</a>";
+                                 + "Page Source </a>";
```

Comment:
@mach6 the fix is done. When the message is empty or null the link text will have page source